paper-server/src/main/java/io/papermc/paper/command/brigadier/PaperCommandSourceStack.java

```diff
@@ -6,9 +6,11 @@
 import net.minecraft.world.phys.Vec3;
 import org.bukkit.Location;
 import org.bukkit.command.CommandSender;
+import org.bukkit.craftbukkit.entity.CraftEntity;
 import org.bukkit.entity.Entity;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
+import org.jspecify.annotations.NonNull;
```

Comment:
Don't mix nullability annotations either convert the class to jspecify or keep using jetbrains one for now